codespell_lib/data/dictionary.txt

```diff
@@ -15336,6 +15356,7 @@ nmae->name
 nned->need
 nnumber->number
 no-overide->no-override
+nocachable->nocacheable
 nodel->model
```

Comment:
I know it's not yours but please also suggest nodal.

Reply:
Done